src/pages/product/components/recommended-products/index.tsx

```diff
@@ -72,8 +70,8 @@ const RecommendedProducts: React.FC = () => {
 				</Box>
 			</StyledRecommendedProductsHeader>
 			<StyledRecommendedProductsSlider>
-				{products &&
-					products
+				{data &&
+					data.products
```

Comment:
I assume `data?.products?.slice` could be used instead of `data && data.products.slice`